languages/clojure/exercises/concept/booleans/src/booleans.clj

```diff
@@ -0,0 +1,18 @@
+(ns booleans)
+
+(defn can-fast-attack? [knight-awake?]
+  (not knight-awake?))
+
+(defn can-spy? [knight? archer? prisoner?]
```

Comment:
@porkostomus I'd like to suggest adding `-awake` to each params, just to keep consistent and clearer

Reply:
Cool, I did that and moved the file to `example.clj` (this one is now just a stub to be presented to the student)